semanticcpg/src/main/scala/io/shiftleft/semanticcpg/language/types/structure/MethodReturn.scala

```diff
@@ -37,4 +37,9 @@ class MethodReturn(raw: GremlinScala[nodes.MethodReturn])
     * */
   def typ: Type =
     new Type(raw.out(EdgeTypes.EVAL_TYPE).cast[nodes.Type])
+
+  def toReturn: NodeSteps[nodes.Return] =
+    new NodeSteps(raw.flatMap { mr =>
+      __(mr.toReturn: _*)
```

Comment:
Why is this not just `raw.flatMap(_.toReturn)`? I anyway approve now but please check this.

Reply:
It will be (closer to) that in the new traversal dsl, but `GremlinScala.flatMap` takes a `MethodReturn => GremlinScala[Return]`, and `MethodReturn.toReturn` returns a `Seq[Return]`, which we need to lift into a `GremlinScala`.